vendor/bat-native-ledger/src/bat/ledger/internal/database/database_activity_info_unittest.cc

```diff
@@ -177,28 +186,4 @@ TEST_F(DatabaseActivityInfoTest, DeleteRecordEmpty) {
   activity_->DeleteRecord("", [](const ledger::Result){});
 }
 
-TEST_F(DatabaseActivityInfoTest, DeleteRecordOk) {
```

Comment:
This removal seems unrelated - was it intentional?

Reply:
reverted